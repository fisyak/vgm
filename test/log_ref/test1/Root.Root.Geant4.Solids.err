Warning in <TGeoManager::Init>: Deleting previous geometry: VGM Root geometry/VGM Root geometry
Warning in <TGeoManager::dtor>: deleting previous geometry: VGM Root geometry/VGM Root geometry
Warning in <TGeoMatrix::dtor>: Registered matrix  was removed
Warning in <TGeoMatrix::dtor>: Registered matrix  was removed
Warning in <TGeoMatrix::dtor>: Registered matrix  was removed
Warning in <TGeoMatrix::dtor>: Registered matrix  was removed
Warning in <TGeoMatrix::dtor>: Registered matrix  was removed
Warning in <TGeoMatrix::dtor>: Registered matrix  was removed
Warning in <TGeoMatrix::dtor>: Registered matrix  was removed
Warning in <TGeoMatrix::dtor>: Registered matrix  was removed
Warning in <TGeoMatrix::dtor>: Registered matrix  was removed
Warning in <TGeoMatrix::dtor>: Registered matrix  was removed
