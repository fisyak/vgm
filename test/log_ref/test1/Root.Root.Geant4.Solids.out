Root geometry will be defined via Root and exported to Geant4

*************************************************************
 Geant4 version Name: geant4-09-02    (19-December-2008)
                      Copyright : Geant4 Collaboration
                      Reference : NIM A 506 (2003), 250-303
                            WWW : http://cern.ch/geant4
*************************************************************

Visualization Manager instantiating...
TestSolids finished
Go to Import
VGM info:   RootGM::Factory::Import started ...
VGM info:   Importing material: 
VGM info:   Material Basic    A=26.98 Z=13 rho=2.7 radlen=8.8751 index=0
VGM info:   Importing element: ALUMINIUM AL from material
VGM info:   Importing material: 
VGM info:   Mixture Air    Aeff=14.607 Zeff=7.3 rho=0.00129 radlen=28405.7 index=1
   Element #0 : N  Z=  7.00 A= 14.01 w= 0.700
   Element #1 : O  Z=  8.00 A= 16.00 w= 0.300
VGM info:   Importing element: NITROGEN N from material
VGM info:   Importing element: OXYGEN O from material
VGM info:   Importing material: 
VGM info:   Mixture Scintillator    Aeff=11.0723 Zeff=5.57354 rho=1.032 radlen=42.132 index=2
   Element #0 : C  Z=  6.00 A= 12.01 w= 0.915 natoms=9
   Element #1 : H  Z=  1.00 A=  1.01 w= 0.085 natoms=10
VGM info:   Importing element: CARBON C from TGeoTable
VGM info:   Importing element: HYDROGEN H from TGeoTable
VGM info:   Importing material: 
VGM info:   Material Uranium    A=235.31 Z=92 rho=13.61 radlen=0.436285 index=3
VGM info:   Importing element: URANIUM U from material
VGM info:   Importing material: 
VGM info:   Material Vacuum    A=0 Z=0 rho=0 radlen=0 index=4
VGM info:   Importing element: VACUUM VACUUM from TGeoTable
VGM info:   Importing medium: 
VGM info:   OBJ: TGeoMedium	Basic	
VGM info:   Importing medium: 
VGM info:   OBJ: TGeoMedium	Air	
VGM info:   Importing medium: 
VGM info:   OBJ: TGeoMedium	Scintillator	
VGM info:   Importing medium: 
VGM info:   OBJ: TGeoMedium	Uranium	
VGM info:   Importing medium: 
VGM info:   OBJ: TGeoMedium	Vacuum	
VGM info:   Importing volume: world
VGM info:   Importing shape: worldS
VGM info:   ImportDaughters for world
VGM info:   Importing volume: box
VGM info:   Importing shape: boxS
VGM info:   ImportDaughters for box
VGM info:   Importing volume: cons
VGM info:   Importing shape: consS
VGM info:   ImportDaughters for cons
VGM info:   Importing volume: eltu
VGM info:   Importing shape: eltuS
VGM info:   ImportDaughters for eltu
VGM info:   Importing volume: para
VGM info:   Importing shape: paraS
VGM info:   ImportDaughters for para
VGM info:   Importing volume: pcone
VGM info:   Importing shape: pconeS
VGM info:   ImportDaughters for pcone
VGM info:   Importing volume: phedra
VGM info:   Importing shape: phedraS
VGM info:   ImportDaughters for phedra
VGM info:   Importing volume: sphere
VGM info:   Importing shape: sphereS
VGM info:   ImportDaughters for sphere
VGM info:   Importing volume: torus
VGM info:   Importing shape: torusS
VGM info:   ImportDaughters for torus
VGM info:   Importing volume: trap
VGM info:   Importing shape: trapS
VGM info:   ImportDaughters for trap
VGM info:   Importing volume: trd
VGM info:   Importing shape: trdS
VGM info:   ImportDaughters for trd
VGM info:   Importing volume: tubs
VGM info:   Importing shape: tubsS
VGM info:   ImportDaughters for tubs
VGM info:   Importing volume: ctubs
VGM info:   Importing shape: ctubsS
VGM info:   ImportDaughters for ctubs
VGM info:   Importing volume: xtru1
VGM info:   Importing shape: TGeoXtru
VGM info:   ImportDaughters for xtru1
VGM info:   Importing volume: xtru2
VGM info:   Importing shape: TGeoXtru
VGM info:   ImportDaughters for xtru2
VGM info:   Import positions: 
VGM info:   0th volume: world  
VGM info:      0th daughter  rtVol = box vgmVol = box
VGM info:      1th daughter  rtVol = cons vgmVol = cons
VGM info:      2th daughter  rtVol = eltu vgmVol = eltu
VGM info:      3th daughter  rtVol = para vgmVol = para
VGM info:      4th daughter  rtVol = pcone vgmVol = pcone
VGM info:      5th daughter  rtVol = phedra vgmVol = phedra
VGM info:      6th daughter  rtVol = sphere vgmVol = sphere
VGM info:      7th daughter  rtVol = torus vgmVol = torus
VGM info:      8th daughter  rtVol = trap vgmVol = trap
VGM info:      9th daughter  rtVol = trd vgmVol = trd
VGM info:      10th daughter  rtVol = tubs vgmVol = tubs
VGM info:      11th daughter  rtVol = ctubs vgmVol = ctubs
VGM info:      12th daughter  rtVol = xtru1 vgmVol = xtru1
VGM info:      13th daughter  rtVol = xtru2 vgmVol = xtru2
VGM info:   1th volume: box  
VGM info:   2th volume: cons  
VGM info:   3th volume: eltu  
VGM info:   4th volume: para  
VGM info:   5th volume: pcone  
VGM info:   6th volume: phedra  
VGM info:   7th volume: sphere  
VGM info:   8th volume: torus  
VGM info:   9th volume: trap  
VGM info:   10th volume: trd  
VGM info:   11th volume: tubs  
VGM info:   12th volume: ctubs  
VGM info:   13th volume: xtru1  
VGM info:   14th volume: xtru2  
VGM info:   
Root_GM_Factory factory solids store: 
   0th solid: Box  "worldS"  hx = 6000mm  hy = 3000mm  hz = 2000mm
   1th solid: Box  "boxS"  hx = 200mm  hy = 600mm  hz = 500mm
   2th solid: Cons  "consS"  rin1 = 100mm  rout1 = 400mm  rin2 = 200mm  rout2 = 600mm  hz = 500mm  sphi = 0deg  dphi = 360deg
   3th solid: Eltu  "eltuS"  dx = 200mm  dy = 300mm  hz = 500mm
   4th solid: Para  "paraS"  hx = 400mm  hy = 600mm  hz = 500mm  alpha = 30deg  theta = 30deg  phi = 30deg
   5th solid: Polycone  "pconeS"  sphi = 0deg  dphi = 360deg  nz = 3
   0th plane:  z = -500mm rin = 100mm rout = 500mm
   1th plane:  z = 100mm rin = 200mm rout = 300mm
   2th plane:  z = 400mm rin = 300mm rout = 600mm
   6th solid: Polygone  "phedraS"  sphi = 0deg  dphi = 360deg  nsides = 4  nz = 3
   0th plane:  z = -500mm rin = 100mm rout = 400mm
   1th plane:  z = 100mm rin = 200mm rout = 200mm
   2th plane:  z = 400mm rin = 300mm rout = 500mm
   7th solid: Sphere  "sphereS"  rin = 200mm  rout = 600mm  sphi = 0deg  dphi = 360deg  stheta = 0deg  dtheta = 180deg
   8th solid: Torus  "torusS"  rin = 200mm  rout = 300mm  rtor = 400mm  sphi = 0deg  dphi = 360deg
   9th solid: Trap  "trapS"  hz = 300mm  theta = 25deg  phi = 35deg  hy1 = 200mm  hx1 = 100mm  hx2 = 150mm  alpha1 = 0deg  hy2 = 200mm  hx3 = 100mm  hx4 = 150mm  alpha2 = 0deg
   10th solid: Trd  "trdS"  hx1 = 200mm  hx2 = 300mm  hy1 = 400mm  hy2 = 500mm  hz = 500mm
   11th solid: Tubs  "tubsS"  rin = 200mm  rout = 400mm  hz = 500mm  sphi = 0deg  dphi = 360deg
   12th solid: Ctubs  "ctubsS"  rin = 200mm  rout = 300mm  hz = 604.9mm  sphi = 330deg  dphi = 280deg  nlow = (0, 0.64, -0.77)  nhigh = (0, 0.09, 0.87)
   13th solid: Extruded  "TGeoXtru"
Polygon,  8 vertices:
  (-300, -300)  (-300, 300)  (300, 300)  (300, -300)  (150, -300)  (150, 150)  (-150, 150)  (-150, -300) mm
Planes:
  z = -400 mm    x0 = -200 mm    y0 = 100 mm    scale= 1.5
  z = 100 mm    x0 = 0 mm    y0 = 0 mm    scale= 0.5
  z = 150 mm    x0 = 0 mm    y0 = 0 mm    scale= 0.7
  z = 400 mm    x0 = 200 mm    y0 = 200 mm    scale= 0.9

   14th solid: Extruded  "TGeoXtru"
Polygon,  8 vertices:
  (-300, -300)  (-300, 300)  (300, 300)  (300, -300)  (150, -300)  (150, 150)  (-150, 150)  (-150, -300) mm
Planes:
  z = -400 mm    x0 = -200 mm    y0 = 100 mm    scale= 1.5
  z = 100 mm    x0 = 0 mm    y0 = 0 mm    scale= 0.5
  z = 100 mm    x0 = 0 mm    y0 = 0 mm    scale= 0.7
  z = 400 mm    x0 = 200 mm    y0 = 200 mm    scale= 0.9

Root_GM_Factory factory volumes store: 
   0th: Volume: "world" material "Basic" solid "Box"  14 daughters: "box"  "cons"  "eltu"  "para"  "pcone"  "phedra"  "sphere"  "torus"  "trap"  "trd"  "tubs"  "ctubs"  "xtru1"  "xtru2"  
   1th: Volume: "box" material "Basic" solid "Box"  0 daughters: 
   2th: Volume: "cons" material "Basic" solid "Cons"  0 daughters: 
   3th: Volume: "eltu" material "Basic" solid "Eltu"  0 daughters: 
   4th: Volume: "para" material "Basic" solid "Para"  0 daughters: 
   5th: Volume: "pcone" material "Basic" solid "Polycone"  0 daughters: 
   6th: Volume: "phedra" material "Basic" solid "Polygone"  0 daughters: 
   7th: Volume: "sphere" material "Basic" solid "Sphere"  0 daughters: 
   8th: Volume: "torus" material "Basic" solid "Torus"  0 daughters: 
   9th: Volume: "trap" material "Basic" solid "Trap"  0 daughters: 
   10th: Volume: "trd" material "Basic" solid "Trd"  0 daughters: 
   11th: Volume: "tubs" material "Basic" solid "Tubs"  0 daughters: 
   12th: Volume: "ctubs" material "Basic" solid "Ctubs"  0 daughters: 
   13th: Volume: "xtru1" material "Basic" solid "Extruded"  0 daughters: 
   14th: Volume: "xtru2" material "Basic" solid "Extruded"  0 daughters: 
VGM info:   RootGM::Factory::Import finished.
Import finished
Go to export
VGM info:   Exporting element: 
VGM info:   Element: "ALUMINIUM"  "AL"  Z = 13  N = 26.98  A = 26.98 g/mole
VGM info:   Exporting element: 
VGM info:   Element: "NITROGEN"  "N"  Z = 7  N = 14.01  A = 14.01 g/mole
VGM info:   Exporting element: 
VGM info:   Element: "OXYGEN"  "O"  Z = 8  N = 16  A = 16 g/mole
VGM info:   Exporting element: 
VGM info:   Element: "CARBON"  "C"  Z = 6  N = 12.0107  A = 12.0107 g/mole
VGM info:   Exporting element: 
VGM info:   Element: "HYDROGEN"  "H"  Z = 1  N = 1.00794  A = 1.00794 g/mole
VGM info:   Exporting element: 
VGM info:   Element: "URANIUM"  "U"  Z = 92  N = 235.31  A = 235.31 g/mole
VGM info:   Exporting element: 
VGM info:   Element: "VACUUM"  "VACUUM"  Z = 0  N = 0  A = 0 g/mole
VGM info:   Exporting material: 
VGM info:   Material: "Basic"  density 2.7 g/cm3  radlen 8.8751 cm  intlen 0 cm  state "Undefined"  temperature 273.15 kelvin  pressure 1 atm
   0th element: Element: "ALUMINIUM"  "AL"  Z = 13  N = 26.98  A = 26.98 g/mole  Elm mass fraction: 100 %
VGM info:   Exporting material: 
VGM info:   Material: "Air"  density 0.00129 g/cm3  radlen 28405.7 cm  intlen 0 cm  state "Undefined"  temperature 273.15 kelvin  pressure 1 atm
   0th element: Element: "NITROGEN"  "N"  Z = 7  N = 14.01  A = 14.01 g/mole  Elm mass fraction: 70 %
   1th element: Element: "OXYGEN"  "O"  Z = 8  N = 16  A = 16 g/mole  Elm mass fraction: 30 %
VGM info:   Exporting material: 
VGM info:   Material: "Scintillator"  density 1.032 g/cm3  radlen 42.132 cm  intlen 0 cm  state "Undefined"  temperature 273.15 kelvin  pressure 1 atm
   0th element: Element: "CARBON"  "C"  Z = 6  N = 12.0107  A = 12.0107 g/mole  Elm mass fraction: 91.4708 %
   1th element: Element: "HYDROGEN"  "H"  Z = 1  N = 1.00794  A = 1.00794 g/mole  Elm mass fraction: 8.52916 %
VGM info:   Exporting material: 
VGM info:   Material: "Uranium"  density 13.61 g/cm3  radlen 0.436285 cm  intlen 0 cm  state "Undefined"  temperature 273.15 kelvin  pressure 1 atm
   0th element: Element: "URANIUM"  "U"  Z = 92  N = 235.31  A = 235.31 g/mole  Elm mass fraction: 100 %
VGM info:   Exporting material: 
VGM info:   Material: "Vacuum"  density 0 g/cm3  radlen 0 cm  intlen 0 cm  state "Gas"  temperature 2.73 kelvin  pressure 2.96077e-23 atm
   0th element: Element: "VACUUM"  "VACUUM"  Z = 0  N = 0  A = 0 g/mole  Elm mass fraction: 100 %
VGM info:   Exporting medium: 
VGM info:   Medium: "Basic"  material Basic g/cm3  parameters:  0  0  0  0  0  0  0  0  0  0  0  0  0  0  0  0  0  0  0  0
VGM info:   Exporting medium: 
VGM info:   Medium: "Air"  material Air g/cm3  parameters:  0  0  0  0  0  0  0  0  0  0  0  0  0  0  0  0  0  0  0  0
VGM info:   Exporting medium: 
VGM info:   Medium: "Scintillator"  material Scintillator g/cm3  parameters:  0  0  0  0  0  0  0  0  0  0  0  0  0  0  0  0  0  0  0  0
VGM info:   Exporting medium: 
VGM info:   Medium: "Uranium"  material Uranium g/cm3  parameters:  0  0  0  0  0  0  0  0  0  0  0  0  0  0  0  0  0  0  0  0
VGM info:   Exporting medium: 
VGM info:   Medium: "Vacuum"  material Vacuum g/cm3  parameters:  0  0  0  0  0  0  0  0  0  0  0  0  0  0  0  0  0  0  0  0
VGM info:   Exporting volume store: 
VGM info:   Exporting solid: 
VGM info:      Box  "worldS"  hx = 6000mm  hy = 3000mm  hz = 2000mm
VGM info:      0th volume: world   material: Basic
VGM info:   Exporting solid: 
VGM info:      Box  "boxS"  hx = 200mm  hy = 600mm  hz = 500mm
VGM info:      1th volume: box   material: Basic
VGM info:   Exporting solid: 
VGM info:      Cons  "consS"  rin1 = 100mm  rout1 = 400mm  rin2 = 200mm  rout2 = 600mm  hz = 500mm  sphi = 0deg  dphi = 360deg
VGM info:      2th volume: cons   material: Basic
VGM info:   Exporting solid: 
VGM info:      Eltu  "eltuS"  dx = 200mm  dy = 300mm  hz = 500mm
VGM info:      3th volume: eltu   material: Basic
VGM info:   Exporting solid: 
VGM info:      Para  "paraS"  hx = 400mm  hy = 600mm  hz = 500mm  alpha = 30deg  theta = 30deg  phi = 30deg
VGM info:      4th volume: para   material: Basic
VGM info:   Exporting solid: 
VGM info:      Polycone  "pconeS"  sphi = 0deg  dphi = 360deg  nz = 3
   0th plane:  z = -500mm rin = 100mm rout = 500mm
   1th plane:  z = 100mm rin = 200mm rout = 300mm
   2th plane:  z = 400mm rin = 300mm rout = 600mm
VGM info:      5th volume: pcone   material: Basic
VGM info:   Exporting solid: 
VGM info:      Polygone  "phedraS"  sphi = 0deg  dphi = 360deg  nsides = 4  nz = 3
   0th plane:  z = -500mm rin = 100mm rout = 400mm
   1th plane:  z = 100mm rin = 200mm rout = 200mm
   2th plane:  z = 400mm rin = 300mm rout = 500mm
VGM info:      6th volume: phedra   material: Basic
VGM info:   Exporting solid: 
VGM info:      Sphere  "sphereS"  rin = 200mm  rout = 600mm  sphi = 0deg  dphi = 360deg  stheta = 0deg  dtheta = 180deg
VGM info:      7th volume: sphere   material: Basic
VGM info:   Exporting solid: 
VGM info:      Torus  "torusS"  rin = 200mm  rout = 300mm  rtor = 400mm  sphi = 0deg  dphi = 360deg
VGM info:      8th volume: torus   material: Basic
VGM info:   Exporting solid: 
VGM info:      Trap  "trapS"  hz = 300mm  theta = 25deg  phi = 35deg  hy1 = 200mm  hx1 = 100mm  hx2 = 150mm  alpha1 = 0deg  hy2 = 200mm  hx3 = 100mm  hx4 = 150mm  alpha2 = 0deg
VGM info:      9th volume: trap   material: Basic
VGM info:   Exporting solid: 
VGM info:      Trd  "trdS"  hx1 = 200mm  hx2 = 300mm  hy1 = 400mm  hy2 = 500mm  hz = 500mm
VGM info:      10th volume: trd   material: Basic
VGM info:   Exporting solid: 
VGM info:      Tubs  "tubsS"  rin = 200mm  rout = 400mm  hz = 500mm  sphi = 0deg  dphi = 360deg
VGM info:      11th volume: tubs   material: Basic
VGM info:   Exporting solid: 
VGM info:      Ctubs  "ctubsS"  rin = 200mm  rout = 300mm  hz = 604.9mm  sphi = 330deg  dphi = 280deg  nlow = (0, 0.64, -0.77)  nhigh = (0, 0.09, 0.87)
VGM info:      12th volume: ctubs   material: Basic
VGM info:   Exporting solid: 
VGM info:      Extruded  "TGeoXtru"
Polygon,  8 vertices:
  (-300, -300)  (-300, 300)  (300, 300)  (300, -300)  (150, -300)  (150, 150)  (-150, 150)  (-150, -300) mm
Planes:
  z = -400 mm    x0 = -200 mm    y0 = 100 mm    scale= 1.5
  z = 100 mm    x0 = 0 mm    y0 = 0 mm    scale= 0.5
  z = 150 mm    x0 = 0 mm    y0 = 0 mm    scale= 0.7
  z = 400 mm    x0 = 200 mm    y0 = 200 mm    scale= 0.9

VGM info:      13th volume: xtru1   material: Basic
VGM info:   Exporting solid: 
VGM info:      Extruded  "TGeoXtru"
Polygon,  8 vertices:
  (-300, -300)  (-300, 300)  (300, 300)  (300, -300)  (150, -300)  (150, 150)  (-150, 150)  (-150, -300) mm
Planes:
  z = -400 mm    x0 = -200 mm    y0 = 100 mm    scale= 1.5
  z = 100 mm    x0 = 0 mm    y0 = 0 mm    scale= 0.5
  z = 100 mm    x0 = 0 mm    y0 = 0 mm    scale= 0.7
  z = 400 mm    x0 = 200 mm    y0 = 200 mm    scale= 0.9

VGM info:      14th volume: xtru2   material: Basic
VGM info:   ExportPlacements for 0th volume world
VGM info:      0th daughter vol = box  simple placement: box
VGM info:      1th daughter vol = cons  simple placement: cons
VGM info:      2th daughter vol = eltu  simple placement: eltu
VGM info:      3th daughter vol = para  simple placement: para
VGM info:      4th daughter vol = pcone  simple placement: pcone
VGM info:      5th daughter vol = phedra  simple placement: phedra
VGM info:      6th daughter vol = sphere  simple placement: sphere
VGM info:      7th daughter vol = torus  simple placement: torus
VGM info:      8th daughter vol = trap  simple placement: trap
VGM info:      9th daughter vol = trd  simple placement: trd
VGM info:      10th daughter vol = tubs  simple placement: tubs
VGM info:      11th daughter vol = ctubs  simple placement: ctubs
VGM info:      12th daughter vol = xtru1  simple placement: xtru1
VGM info:      13th daughter vol = xtru2  simple placement: xtru2
VGM info:   ExportPlacements for 1th volume box
VGM info:   ExportPlacements for 2th volume cons
VGM info:   ExportPlacements for 3th volume eltu
VGM info:   ExportPlacements for 4th volume para
VGM info:   ExportPlacements for 5th volume pcone
VGM info:   ExportPlacements for 6th volume phedra
VGM info:   ExportPlacements for 7th volume sphere
VGM info:   ExportPlacements for 8th volume torus
VGM info:   ExportPlacements for 9th volume trap
VGM info:   ExportPlacements for 10th volume trd
VGM info:   ExportPlacements for 11th volume tubs
VGM info:   ExportPlacements for 12th volume ctubs
VGM info:   ExportPlacements for 13th volume xtru1
VGM info:   ExportPlacements for 14th volume xtru2
Export finished
Go to save Root geometry
Visualization Manager initialising...
Registering graphics systems...

You have successfully registered the following graphics systems.
Current available graphics systems are:
  ASCIITree (ATree)
  DAWNFILE (DAWNFILE)
  G4HepRep (HepRepXML)
  G4HepRepFile (HepRepFile)
  RayTracer (RayTracer)
  VRML1FILE (VRML1FILE)
  VRML2FILE (VRML2FILE)
  FukuiRenderer (DAWN)
  OpenGLImmediateX (OGLIX)
  OpenGLStoredX (OGLSX)
  OpenGLImmediateXm (OGLIXm)
  OpenGLStoredXm (OGLSXm)
  OpenInventorXt (OIX)
  VRML1 (VRML1)
  VRML2 (VRML2)

Registering model factories...

You have successfully registered the following model factories.
Registered model factories:
  generic
  drawByCharge
  drawByParticleID
  drawByOriginVolume
  drawByAttribute

Registered filter factories:
  chargeFilter
  particleFilter
  originVolumeFilter
  attributeFilter

Graphics systems deleted.
Visualization Manager deleting...
